src/components/TableFilter/index.tsx

```diff
@@ -50,7 +50,7 @@ const GlobalClientFilter = ({
           // Currently only client-side filtering - updates search filter onChange
           setGlobalFilter(e.target.value);
         }}
-        value={globalFilter}
+        value={globalFilter ?? ''}
```

Comment:
@ClayBenson94 this one liner addressed what you were asking for 🤣


https://github.com/user-attachments/assets/25f02191-46ab-467d-aa4c-934cfa3c6e10